docs/content-onboarding/onboarding.md

```diff
@@ -136,10 +136,14 @@ Host github.com
 * Enter `Title` of your choice (e.g. `ssh-key-TUC`).
 * Set `Key Type` as `Authentication Key`.
 * Paste your public key from the file that was generated (`C:\Users\...\.ssh\id_ed25519.pub`) into the `Key` field.
-* Incase you are not sure what the key is, open a command prompt and type
+* Incase you are unsure of what the key is, open a command prompt and type
 ```
 cat ~/.ssh/id_ed25519.pub
 ```
+* If that doesn't work, open Windows PowerShell and type
+```
+Get-Content ~\.ssh\id_ed25519.pub
+```
```

Comment:
A better way to describe it may be to tell Linux users to use `cat` and Windows users to use `Get-Content`.
But this is fine so far.

(The command for Windows cmd is `type`, btw, but cmd can't resolve the `~` to `%USERPROFILE%`.)